src/components/components/ebay-notice-base/index.marko

```diff
@@ -51,6 +54,12 @@ $ var prefixClass = input.prefixClass;
                     class=input.iconClass
                 />
             </else-if>
+            <else-if(status === 'education' && input.type === 'section')>
+                <${input.educationIcon || LightBulbIcon}
+                    a11y-variant='label'
```

Comment:
I don't think these attributes will ever reach the icon if it is passed on as an attribute tag as you have in `with-icon.marko`. We could potentially have people pass in the icon that was imported instead, like
```marko
import TheVaultIcon from '<ebay-the-ebay-vault-24-fit-icon>'
<ebay-section-notice status="education" icon=TheVaultIcon />
```
but I'm not sure if that is best practice for class API.